kate/recovery/Cargo.toml

```diff
@@ -5,6 +5,7 @@ authors = ["Denis Ermolin <denis.ermolin@matic.network>"]
 edition = "2018"
 
 [dependencies]
+anyhow = "1.0.41"
```

Comment:
Can we replace `anyhow` by `thiserror`?
`thiserror` is better for libraries, and allows you to make a proper error handling when you use the library.

Reply:
Done.